config/routes.rb

```diff
@@ -16,4 +16,8 @@
   get  '/calls/new_connection',           to: 'calls#new_connection'
   post '/calls/connection_gather_prompt', to: 'calls#connection_gather_prompt'
   post '/calls/connection_gather',        to: 'calls#connection_gather'
+
+  require 'sidekiq/api'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.